### PR TITLE
Implement Kilopascal and Megapascal and write tests

### PR DESCRIPTION
**Pull request by arubacao**
For #9 

Cheers

### CODE REVIEW

**Comment by arubacao** [comment]
Tests fail due to composer.lock. 
See https://github.com/jordanbrauer/unit-converter/pull/30

File: src/UnitConverter/Unit/Pressure/Kilopascal.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+declare(strict_types = 1);
+
+namespace UnitConverter\Unit\Pressure;
+
+use UnitConverter\Measure;
+use UnitConverter\Unit\{ AbstractUnit, UnitInterface };
+
+/**
+ * Pascal unit data class.
```
Comment (jordanbrauer) [inline src/UnitConverter/Unit/Pressure/Kilopascal.php]: Correct docblock to reflect unit concrete class unit name :)

File: src/UnitConverter/Unit/Pressure/Megapascal.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+declare(strict_types = 1);
+
+namespace UnitConverter\Unit\Pressure;
+
+use UnitConverter\Measure;
+use UnitConverter\Unit\{ AbstractUnit, UnitInterface };
+
+/**
+ * Pascal unit data class.
```
Comment (jordanbrauer) [inline src/UnitConverter/Unit/Pressure/Megapascal.php]: Same as the previous review - correct the docblock to reflect the class name.

**Comment by jordanbrauer** [comment]
Thanks a bunch @arubacao for your submission!

I've requested some very minor (rather nit-picky) changes before I can pull this into develop branch. :)

**Pull request merged**